### ISSUE TITLE
Input field IDs are not unique (all input fields use "field-input-form-view")

### ISSUE THREAD

**Issue by tisto** [kitconcept/volto-form-builder#50]
1) as admin
2) create a new form with two input fields
-> both input fields use the same id "field-input-form-view" which breaks CSS (IDs should be unique) as well as accessibility (label for is useless if it points to two elements)
